content/en/docs/setup/independent/install-kubeadm.md

```diff
@@ -234,23 +234,17 @@ kubeadm to tell it what to do.
 
 ## Configure cgroup driver used by kubelet on Master Node
 
-Make sure that the cgroup driver used by kubelet is the same as the one used by Docker. Verify that your Docker cgroup driver matches the kubelet config:
+When using Docker, kubeadm will automatically detect the cgroup driver for the kubelet
+and set it in the `/var/lib/kubelet/kubeadm-flags.env` file during runtime.
 
-```bash
-docker info | grep -i cgroup
-cat /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
-```
-
-If the Docker cgroup driver and the kubelet config don't match, change the kubelet config to match the Docker cgroup driver. The
-flag you need to change is `--cgroup-driver`. If it's already set, you can update like so:
-
-```bash
-sed -i "s/cgroup-driver=systemd/cgroup-driver=cgroupfs/g" /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
-```
+If you are using a different CRI, you have to modify the the drop-in file
+`/etc/systemd/system/kubelet.service.d/10-kubeadm.conf`, by adding
```

Comment:
It is not good to modify configuration file that comes from package. Better to mention on how to add user's drop-in with override of cgroups-driver argument.